voice/methods/calls/deleteCallsCallIdRecordingsRecordingId.md

```diff
@@ -0,0 +1,37 @@
+{% method %}
+
+## Delete Recording
+Delete both the metadata and the media of a specific recording.
+
+### Request URL
+
+<code class="post">DELETE</code>`https://voice.bandwidth.com/api/v2/accounts/{accountId}/calls/{callId}/recordings/{recordingId}`
+
+#### Basic Authentication
+
+Bandwidth's Voice API leverages Basic Authentication with your Dashboard API Credentials. Read more about how Bandwidth secures endpoints in the [Security & Credentials](../../../guides/accountCredentials.md) document.
```

Comment:
Confused why this is in here. The other endpoint details don't have this do they?

Reply:
I think they all have it. Tons of duplicate code :disappointed:

Reply:
Actually, this was missing from the pause/resume endpoint that was previously merged, so I added it there.